### ISSUE TITLE
Scene summary missing a period.

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#108]
@zepumph, I noticed we are missing a period at the end of a sentence in the introductory paragraph (scene summary).
<img width="467" alt="scen-summary-missing-period" src="https://user-images.githubusercontent.com/1902116/53866204-493e5200-3ff1-11e9-87b2-1a111b4792a5.png">
